src/components/common/FileUploadPreview.vue

```diff
@@ -85,6 +86,11 @@ export default class FileUploadPreview extends Mixins(DocumentMixin) {
     await this.computeEmitFileValidity(this.fileUpload)
   }
 
+  /** Can be called externally to open the file selection dialog. */
+  public clickFileInput (): void {
+    document.getElementById('file-input').click()
+  }
```

Comment:
This allows the user to click the Add button (in the parent) and just see the file dialog right away.